model/database_functions.ts

```diff
@@ -1,15 +1,37 @@
 import { database } from "./createDatabase";
 import { Journey, Trip, Location } from "./model";
 
-const createJourney = async (title: string): Promise<Journey> => {
+export const createJourney = async (title: string): Promise<Journey> => {
   return database.write(async () => {
     return database.get<Journey>("journeys").create((journey) => {
       journey.title = title;
+      journey.isActive = false;
     });
   });
 };
 
-const createTrip = async (journeyId: string, title: string): Promise<Trip> => {
+export const setJourneyActive = async (journeyId: string) => {
+  database.write(async () => {
```

Comment:
```suggestion
  await database.write(async () => {
```

Ohne awaits werden die Promises nicht aufgelöst.